config/application.rb

```diff
@@ -24,7 +24,7 @@ class Application < Rails::Application
     config.load_defaults 7.0    
     config.active_storage.variant_processor = :mini_magick
     config.generators.system_tests = nil
-    config.active_job.queue_adapter = :sidekiq
+    # config.active_job.queue_adapter = :sidekiq
```

Comment:
What happened?

Reply:
Redis is giving errors when trying to seed DB with user pictures in development with Faker. That's why I comment it seems like I forgot to uncomment it